fivox/eventSource.cpp

```diff
@@ -166,18 +163,10 @@ void EventSource::add( const Event& event )
     _impl->events.push_back( event );
 }
 
-void EventSource::update( const size_t index, float value )
+void EventSource::update( const size_t index, const float value )
```

Comment:
This now updates only the value. Other loaders might change the radius too. Ideally make event.position const and have an operator[] (eg ```source[index].radius = newRadius;``` or rename to updateValue?